### PR TITLE
Frontend/feature/191 add frontend deploy GitHub actions

### PR DESCRIPTION
**Pull request by boraykasap**

### CODE REVIEW

File: .github/workflows/frontend.yml
Comment (mmtftr) [inline .github/workflows/frontend.yml:1]: why is the frontend workflow deleted? please restore this file

File: backend/src/main/resources/application.properties [hunk 1]
```diff
@@ -5,7 +5,7 @@ jwt.signing.key=413F4428472B4B6250655368566D5970337336763979244226452948404D6351
 spring.application.name=cuisines
 spring.datasource.url=jdbc:mysql://localhost:3306/cuisines-test?createDatabaseIfNotExist=true
 spring.datasource.username=root
-spring.datasource.password=adminadmin
+spring.datasource.password=admin
```
Comment (mmtftr) [inline backend/src/main/resources/application.properties:8]: why do we need this change?

File: .github/workflows/frontend_deploy_staging.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      - staging
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Authorize with doctl
```
Comment (mmtftr) [inline .github/workflows/frontend_deploy_staging.yml:18]: can we instead use actions/setup-doctl as mentioned [here](https://docs.digitalocean.com/products/kubernetes/how-to/deploy-using-github-actions/#specify-steps) on step 2

**Review by mmtftr — CHANGES_REQUESTED** [review]

File: backend/src/main/resources/application.properties (hunk 1 above)
Comment (boraykasap) [reply backend/src/main/resources/application.properties:8]: admin is actually the default one. adminadmin was wrong.

File: .github/workflows/frontend.yml
Comment (boraykasap) [reply .github/workflows/frontend.yml:1]: there was 2 of them with different names. i deleted one.

File: .github/workflows/frontend_deploy_staging.yml (hunk 2 above)
Comment (boraykasap) [reply .github/workflows/frontend_deploy_staging.yml:18]: taken into account

**Comment by mmtftr** [comment]
can we remove on pull request from the frontend test? Just push is enough

File: .github/workflows/frontend_deploy_staging.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      - completed
+    branches:
+      - staging
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: digitalocean/action-doctl@v2
```
Comment (mmtftr) [inline .github/workflows/frontend_deploy_staging.yml:16]: please revert this to checkout action

File: .github/workflows/frontend_deploy_staging.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+      - name: Authorize with doctl
+        run: echo ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }} | doctl auth init --context github-actions
```
Comment (mmtftr) [inline .github/workflows/frontend_deploy_staging.yml:19]: if possible this should be the action-doctl using our digitalocean access token

**Review by mmtftr — CHANGES_REQUESTED** [review]

**Comment by EnesBaserr** [comment]
Closed due to new term.

**Pull request closed without merging**